core/src/database/proxy_backend/soci-proxy.h

```diff
@@ -59,6 +59,7 @@
 #   define SOCI_OVERRIDE override
 #endif
 
+#define SOCI_PROXY_DEBUG
```

Comment:
I’m not sure whether it’s on purpose, but this will make the block below always gets substituted to the same `SP_PRINT` definition.

Reply:
Oh god yes, I remove that